app/models/pension/charges/LifetimeAllowance.scala

```diff
@@ -24,24 +24,27 @@ import utils.DecryptorInstances.bigDecimalDecryptor
 import utils.EncryptableSyntax.EncryptableOps
 import utils.EncryptorInstances.bigDecimalEncryptor
 
-case class LifetimeAllowance(amount: BigDecimal, taxPaid: BigDecimal) {
+case class LifetimeAllowance(
+                              amount: Option[BigDecimal],
+                              taxPaid: Option[BigDecimal]
+                            ) {
 
   def encrypted()(implicit secureGCMCipher: SecureGCMCipher, textAndKey: TextAndKey): EncryptedLifetimeAllowance =
     EncryptedLifetimeAllowance(
-      amount = amount.encrypted,
-      taxPaid = taxPaid.encrypted
+      amount = amount.map(_.encrypted),
+      taxPaid = taxPaid.map(_.encrypted)
     )
 }
 
 object LifetimeAllowance {
   implicit val format: OFormat[LifetimeAllowance] = Json.format[LifetimeAllowance]
 }
 
-case class EncryptedLifetimeAllowance(amount: EncryptedValue, taxPaid: EncryptedValue) {
+case class EncryptedLifetimeAllowance(amount: Option[EncryptedValue], taxPaid: Option[EncryptedValue]) {
```

Comment:
Could you have both values default to 'None'.

Reply:
Set the default values to None

Reply:
Confirmed.